src/Canvas/Pixel.elm

```diff
@@ -0,0 +1,227 @@
+module Canvas.Pixel
+    exposing
+        ( put
+        , putMany
+        , line
+        , rectangle
+        , bezier
+        )
+
+import Canvas exposing (Canvas, Size, Point, DrawOp(..))
+import Color exposing (Color)
+import Array exposing (Array)
+
+
+put : Color -> Point -> DrawOp
```

Comment:
Re `put`, I just made a commit changing those. I made a new one called `putTuple`, because `putMany` needs a helper function to turn `(Color, Point)` to `Color -> Point -> ...`